schemas/apps.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$id": "https://raw.githubusercontent.com/aiidalab/aiidalab-registry/v1/schemas/apps.schema.json",
+    "$ref": "#/definitions/Welcome",
+    "definitions": {
+        "Welcome": {
+            "type": "object",
+            "propertyNames": {
+              "pattern": "^[A-Za-z][A-Za-z0-9-_]*$"
+            },
+            "additionalProperties": false,
+            "patternProperties": {
+                ".*": {"$ref": "#/definitions/App"}
+            },
+            "title": "Schema for the AiiDAlab apps registry."
+        },
+        "App": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "git_url": {
+                    "type": "string",
+                    "format": "uri",
+                    "qt-uri-protocols": [
+                        "https"
+                    ],
+                    "qt-uri-extensions": [
+                        ".git"
+                    ]
+                },
+                "meta_url": {
+                    "type": "string",
+                    "format": "uri",
+                    "qt-uri-protocols": [
+                        "https"
+                    ],
+                    "qt-uri-extensions": [
+                        ".json"
+                    ]
+                },
+                "categories": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                }
+            },
+            "required": [
+                "categories",
```

Comment:
Not required, at least not according to the README.
If it _is_ now required, then please also update the README.
```suggestion
```

Reply:
You're correct.